src/camera/CircularCamera.ts

```diff
@@ -19,7 +20,68 @@ class CircularCamera extends Camera {
     };
   } = {};
 
-  public get circularEnabled() { return this._circularEnabled; }
+  public get controlParams() { return { range: this._range, position: this._position, circular: this._circularEnabled }; }
+
+  public getPrevAnchor(anchor: AnchorPoint): AnchorPoint | null {
```

Comment:
When the return value of a method is an object, null is often returned when there is no object to return. How about creating an appropriate type and using it?

Reply:
What exactly is the appropriate type you mean?
Is it like `NullPoint`? or `type A = AnchorPoint | null`?

Reply:
This feeling? `type Nullable<T> = T | null;`